recipes/px4tools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "px4tools" %}
+{% set version = "0.4.5" %}
+{% set sha256 = "4c6cf9677961902d439d25911f40b95c36a8ccaa43f9be8ce92cc62ed0a7c7e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/dronecrew/px4tools/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+
+  number: 0
+
+  entry_points:
+    - px42csv=px4tools.px42csv:main
+
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Would suggest cutting down on the extra spacing. Following the order `number`, `script`, `entry_points` should help.